HW_5/Task2.py

```diff
@@ -0,0 +1,11 @@
+#Задача 28: Напишите рекурсивную функцию sum(a, b), возвращающую сумму двух целых неотрицательных чисел. Из всех арифметических операций допускаются только +1 и -1. Также нельзя использовать циклы.
+
+
+def sum(a, b):
+    a += 1
+    b -= 1
+    if b > 0:
+        return sum(a, b)
+    else:
+        return a
+print(sum(6, 1))
```

Comment:
Попробуйте так. Будет сюрприз
```suggestion
print(sum(6, 0))
```